src/routes/admin-portal/users/add/+page.svelte

```diff
@@ -7,12 +7,13 @@
 
 	/** Add a new user to the database and the auth system */
 	async function AddUser() {
+		let loweredEmail = email.toLowerCase()
```

Comment:
```suggestion
		const loweredEmail = email.toLowerCase()
```
NO LET'S FOR IMMUTABLE VARS!

Reply:
i really don't care

Reply:
![image](https://github.com/user-attachments/assets/3a0b6608-7eea-4661-8c38-25fb0b2afcc0)
es-lint does